src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -148,6 +148,13 @@ export default class DefaultAudioVideoController implements AudioVideoController
     return this._mediaStreamBroker;
   }
 
+  async getRTCPeerConnectionStats(selector?: MediaStreamTrack): Promise<RTCStatsReport> {
+    if (!this.rtcPeerConnection) {
+      return null;
+    }
+    return this.rtcPeerConnection.getStats(selector);
```

Comment:
In DefaultStatsCollector, `getStats` takes in success and failure callbacks for Chromium WebRTC instead of using a Promise.
https://github.com/aws/amazon-chime-sdk-js/blob/master/src/statscollector/DefaultStatsCollector.ts#L447

Could you make sure supported browser versions have a promise-based `getStats`?

Here, we just return

Reply:
That is what I thought at first but after talking to consumers, they do not need the legacy stats from Chrome. Chome currently also supports the new getStats that return Promise.

Reply:
Confirmed with Trung that Chrome 58 started to support a promise-based callback